File: code/stable/gooltest/julia/NameGenTest/NameGenTest.jl
module NameGenTest

function helper(temp::Array{Int64})
    result = Int64[]
    
    result = temp[2:3]
    
    @assert length(result) == 2 "Result list should have 2 elements after slicing."
end

global temp = [1, 2, 3]
global result = Int64[]

global result = temp[2:3]

@assert length(result) == 2 "Result list should have 2 elements after slicing."

@assert result[1] == 2 "First element of result should be 2."

end

File: code/stable/gooltest/python/NameGenTest/NameGenTest.py
def helper(temp):
    result = []
    
    result = temp[1:3:]
    
    assert len(result) == 2, "Result list should have 2 elements after slicing."

temp = [1, 2, 3]
result = []

result = temp[1:3:]

assert len(result) == 2, "Result list should have 2 elements after slicing."

assert result[0] == 2, "First element of result should be 2."
